validator/src/main/java/com/amazon/aoc/validators/TraceValidator.java

```diff
@@ -136,7 +136,7 @@ private Map<String, Object> getTrace() throws Exception {
     // Looking for trace generated by /client-call is different from the others because the API call is made by the sample app internally,
     // and not through external callers
     if (validationConfig.getHttpPath().contains("client-call")) {
-      traceFilter += " AND annotation.aws_local_service = \"local-root-client-call\"";
+      traceFilter += " AND (annotation.aws_local_service = \"local-root-client-call\" OR annotation.aws_local_service = \"local-root-client-call:80\")" ;
```

Comment:
nit: You can use regex here instead

Reply:
Actually this way works and I don't want import more complexity? As long as this way is still acceptable